newsfragments/1017.doc.rst

```diff
@@ -0,0 +1 @@
+To help any user reading through Trio's function implementations, start using public names (not _core) whenever possible.  This may not always be possible due to dependencies but should be the general practice.  Two examples of where it is difficult to implement are _sync.py and _threads.py, due to Windows dependencies.  
```

Comment:
This text goes into the release notes. The target audience is, someone who just downloaded the latest trio release and is wondering what's new. I think this is probably going into more detail than people usually want here – they're not digging into the code, they just want to know if anything has happened that they should care about :-). Also, each entry is linked back to the issue tracker (that's what the `1017` in the filename does), so people who do care can click through to get all the details. So I'd drop the last two sentences here.

Reply:
Okay, that makes a lot of sense.  Thanks!